.ruby-version

```diff
@@ -0,0 +1 @@
+2.6.9 # keep at minimum required ruby version
```

Comment:
for those using `rvm` would be great not to add that comments in this file.
just stick with the version.

also, 2.7.5 is probably a better default although its not the minimum required version

Reply:
How does RVM handle the comment? Does it simply not work?

We set it to the minimum required version on purpose to ensure that during local development no newer language features are being made use of.

Reply:
I don't think the comment takes into any real effect, the version that is mentioned in `.ruby_version` would be the source of truth, also, if you want your `bundler` to through error if local ruby is not matching with specification, adding `ruby 2.7.5` over there also fixes the issue. Otherwise, changing the directory to your project with a mismatching ruby version would result a warning from RVM anyway.

Reply:
@jfeltesse-mdsol like a boss

<img width="770" alt="image" src="https://user-images.githubusercontent.com/1864520/153991815-9a7a5a19-b0fa-4f34-8b85-987ab3fddf5b.png">

Reply:
@dsengupta-mdsol I think we're talking about different things. To your first point, well, @mjobin-mdsol just showed it does break RVM. TIL. To your second point about adding some dynamic reading of the `.ruby-version` file so it's processed by bundler through the Gemfile, that's a bad idea as I hinted at on one of my comments in your PR. The reason it's a bad idea is that doing so makes bundler enforce an *exact* ruby version which is a real pain once you touch multiple repos from different teams. I'm not going to install say `2.7.4` when I have `2.7.5` already installed and all I need to do is a quick PR on a repo I don't own. All the version managers provide a `shell` command to override the ruby version on a per shell basis and having bundler enforce the exact version renders that very useful feature useless.

Reply:
removed in 7319fe3

Reply:
@jfeltesse-mdsol  @ykitamura-mdsol Thank you ! 

About having bundler enforce the ruby versions. I have experienced with this. It was useful when I had multiple apps hosted on the same server, and we realized later that some apps were running on a different version than expected.

With the docker setup we have at Medidata, there isn't a huge benefit for a hard lock.

Or even a softer, as bundler support `ruby '~> 2.7.3'` for example.
And it has the benefit to remove the `.ruby-version` file entirely, since (at least  rvm) will autoload it from the gemfile.

I still use it for personal project, so I can't be against it, but there isn't a huge value for us, in my opinion.

Definitely a huge NO for gems, though.